assets/js/components/BatchEdit/About/CoreMetadata.jsx

```diff
@@ -82,18 +81,24 @@ const BatchEditAboutCoreMetadata = ({ ...restProps }) => {
       </div>
       <div className="column is-half">
         <UIFormField label="Rights Statement">
-          <UIFormSelect
-            isReactHookForm
-            name="rightsStatement"
-            label="Rights Statement"
-            options={
-              codeLists.rightsStatementData
-                ? codeLists.rightsStatementData.codeList
-                : []
-            }
-            data-testid="rights-statement"
-            showHelper
-          />
+          <div className="select" data-testid="rights-statement">
+            <select name="rightsStatement" ref={register()}>
+              <option value="">-- Select --</option>
+              {codeLists.rightsStatementData &&
+                codeLists.rightsStatementData.codeList.map((item) => (
+                  <option
+                    key={item.id}
+                    value={JSON.stringify({
+                      id: item.id,
+                      scheme: "RIGHTS_STATEMENT",
+                      label: item.label,
+                    })}
+                  >
+                    {item.label}
+                  </option>
+                ))}
+            </select>
+          </div>
         </UIFormField>
```

Comment:
Can you explain how this fixes the bug?   Would the same effect be achieved by prepping an `options` array and feeding it into `UIFormSelect`, or is there something with `UIFormSelect` that gets in the way?

Reply:
For `UIFormSelect`, there is not an easy way to mention custom values for options, like in the case of codelists, we need to provide ID: RIGHTS_STATEMENT and also clear out label field that we use in batchConfirmation screen.